### ISSUE TITLE
bru2bids will not execute if no functional scans are present

### ISSUE THREAD

**Issue by TheChymera** [IBT-FMI/SAMRI#59]
```
chymera@clusterhost ~/ni_data/ofM.dr $ SAMRI bru2bids -s '{"acquisition":["TurboRARE","TurboRAREhd"]}' . --measurements *6469*                                                                                                    
Traceback (most recent call last):                      
  File "/home/chymera/.python_develop/SAMRI", line 11, in <module>                                               
    load_entry_point('SAMRI', 'console_scripts', 'SAMRI')()                                                      
  File "/home/chymera/src/SAMRI/samri/cli.py", line 9, in main                                                   
    argh.dispatch_commands([diagnose, bru2bids])        
  File "/usr/lib64/python2.7/site-packages/argh/dispatching.py", line 328, in dispatch_commands                  
    dispatch(parser, *args, **kwargs)                   
  File "/usr/lib64/python2.7/site-packages/argh/dispatching.py", line 174, in dispatch                           
    for line in lines:                                  
  File "/usr/lib64/python2.7/site-packages/argh/dispatching.py", line 277, in _execute_command                   
    for line in result:                                 
  File "/usr/lib64/python2.7/site-packages/argh/dispatching.py", line 260, in _call                              
    result = function(*positional, **keywords)          
  File "/home/chymera/src/SAMRI/samri/pipelines/reposit.py", line 107, in bru2bids                               
    get_f_scan.iterables = ("scan_type", functional_scan_types)                                                  
UnboundLocalError: local variable 'functional_scan_types' referenced before assignment 
```

The logic needs to be updated so that the workflow graph gets constructed even in the absence of a `functional_scan_types` list.

**Comment by grandjeanlab** [comment]
Any chance to have an update on this soon? I run into the issue for my brain volumetry scans.

**Comment by TheChymera** [comment]
attempted fix in e6df99d7c939e1227d276486ebcd422d2e3b9fcc . @joanesg try it out!

**Comment by grandjeanlab** [comment]
I tested it. It seems issue is resolved. Thx @TheChymera
